### PR TITLE
fix translation fr

### PR DESCRIPTION
**Pull request by oallain**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Fixed issue   | 
| License       | MIT

### CODE REVIEW

**Review by TheGrimmChester — APPROVED** [review]

File: src/Controller/CommandExecuteImmediateController.php [hunk 1]
```diff
@@ -8,7 +8,7 @@
 use Symfony\Component\HttpFoundation\Response;
 use Synolia\SyliusSchedulerCommandPlugin\Service\ExecuteScheduleCommand;
 
-class ScheduledCommandExecuteImmediateController extends AbstractController
+class CommandExecuteImmediateController extends AbstractController
```
Comment (Jibbarth) [inline src/Controller/CommandExecuteImmediateController.php:11]: Be careful, this change may be a Breaking Change

Comment (oallain) [reply src/Controller/CommandExecuteImmediateController.php:11]: Yes, what do you advise?

Comment (oallain) [reply src/Controller/CommandExecuteImmediateController.php]: After @Jibbarth 's good remark, I prefer to change the PhpMD rules so as not to introduce a BC break.

**Review by TheGrimmChester — APPROVED** [review]

**Comment by oallain** [comment]
Sylius master builds will be fixed in #28

**Pull request merged**